cmd/root.go

```diff
@@ -222,6 +239,39 @@ func (authArgs *authArgs) getClient() (armhelpers.AKSEngineClient, error) {
 	return client, nil
 }
 
+func (authArgs *authArgs) getazsClient() (armhelpers.AKSEngineClient, error) {
```

Comment:
Let's rename this func to `getAzureStackClient` so it can't be accidentally mistaken for `getazClient` by humans. :)

Reply:
sure :)